hovercraft/local_directives/example.__init__.py

```diff
@@ -0,0 +1,259 @@
+"""
```

Comment:
What kind of filename is that?

Reply:
Sorry, I don't understand. It is exactly what it says on the label,  an example of an __init__.py to be used as a base for making new directives. My aim was to make it as obvious as possible for people who aren't experienced coders (such as myself). 

I'm happy to change it if there is a better option.

Reply:
Plugins should reasonably not be called `pluginname.__init__.py,` so  that doesn't make any sense. It should be plugins/example_plugin/__init__.py in that case.